packages/design-system/src/components/slider/types.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  NumberFieldProps as AriaNumberFieldProps,
+  SliderRenderProps as RACSliderRenderProps,
+  SliderProps as RACSliderProps,
+  SliderThumbProps as RACSliderThumbProps,
+  SliderThumbRenderProps as RACSliderThumbRenderProps,
+  SliderTrackRenderProps as RACSliderTrackRenderProps,
+  SliderTrackProps as RACSliderTrackProps,
+  SliderOutputProps as RACSliderOutputProps,
+} from 'react-aria-components';
+import type { RenderPropsChildren } from '../../types';
+import type { PartialDeep } from 'type-fest';
+import type { InputClassNames, InputRenderProps } from '../input';
+import type { GroupClassNames } from '../group';
+
+export type SliderClassNames = PartialDeep<{
+  slider: {
+    container: string;
+    label: string;
+    min: string;
+    max: string;
+  };
+  track: {
+    container: string;
+    track: string; // TODO: adjust the styles to use this class
+  };
+  thumb: {
+    container: string;
+    thumb: string;
+  };
+  input: InputClassNames;
+  output: {
+    container: string;
+  };
+  group: GroupClassNames;
+}>;
+
+export type LabelAlignment = 'top' | 'left';
```

Comment:
Because of the addition of supporting the vertical orientation, I think you're going to want to change this to being something like `'stacked' | 'inline'`